### ISSUE TITLE
Add Portal rating

### ISSUE THREAD

**Issue by prayagd** [pendulum-chain/portal#475]

**Comment by prayagd** [comment]
@ebma @Sharqiewicz anything unclear about the above? is it possible to capture the data on excel sheet?

Marked it as high priority as it is one, once estimated i would request to work on this asap.

**Comment by prayagd** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Pendulum-6408a0365cda59c57d39bee5/issues/pendulum-chain/portal/475?planning-poker) with Zenhub @ebma @Sharqiewicz

**Comment by ebma** [comment]
> If the user closes the pop-up without providing rating, show the pop-up for next successful action.

Isn't this a bit too pushy? I would be annoyed if the app was asking me for feedback all the time, even if I manually close the dialog. I think if a user closes the dialog and actively decides not to provide feedback, we should also not prompt again for at least a week, or maybe treat it like the default case of 15 days. 

> Following info to be captured in a excel file/database

A database is definitely overkill for this so let's rather try to store it differently. While I've never tried it myself, I guess we can send that data to a [Google Form](https://developers.google.com/forms/api/guides).

**Comment by prayagd** [comment]
@Sharqiewicz please add you estimates

**Comment by prayagd** [comment]
> Isn't this a bit too pushy? I would be annoyed if the app was asking me for feedback all the time, even if I manually close the dialog. I think if a user closes the dialog and actively decides not to provide feedback, we should also not prompt again for at least a week, or maybe treat it like the default case of 15 days.

Fair point, made changes in the ticket

**Comment by prayagd** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Pendulum-6408a0365cda59c57d39bee5/issues/pendulum-chain/portal/475?planning-poker) with Zenhub @TorstenStueber

**Comment by TorstenStueber** [comment]
We would need some backend for this, I think we should not directly push to the Google Form from the frontend, otherwise we expose the credentials.

**Comment by Sharqiewicz** [comment]
I think we can use Netlify Functions along with Google Sheets. It won't expose credentials. WDYT? @TorstenStueber @ebma

**Comment by TorstenStueber** [comment]
Yes, Lambda or Netlify functions sounds good. This can be setup and managed by DevOps.

**Comment by Sharqiewicz** [comment]
The code can be copied from **Vortex**
